setup.py

```diff
@@ -6,7 +6,7 @@
       long_description = f.read()
 
 setup(name='pipelinewise-tap-postgres',
-      version='1.0.9',
+      version='1.1.0',
```

Comment:
Just wondering, are we following "major.minor.patch" versioning pattern?

Reply:
I changed the psycopg2 library version ... and thought this might be a valid reason for 1.1.0